jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java

```diff
@@ -286,6 +296,19 @@ private void validateParameters() throws MojoFailureException {
         throw new MojoFailureException("Invalid configuration parameters");
       }
     }
+    // Validates 'imageFormat'
+    try {
+      imageFormatToEnum = ImageFormat.valueOf(imageFormat);
```

Comment:
We probably want to have this be more local to the place it is used and not as a field (ie. perform the check for if `imageFormat` is part of the enum (therefore not depending on catching an `IllegalArgumentException`), but call `ImageFormat.valueOf` above where the configuration is set on the `BuildConfiguration`)